repositories/src/test/scala/cromwell/pipeline/datastorage/dao/repository/UserRepositoryTest.scala

```diff
@@ -69,9 +69,7 @@ class UserRepositoryTest extends AsyncWordSpec with Matchers with BeforeAndAfter
           dummyUser.copy(email = "updated@email.com", firstName = "updatedFName", lastName = "updatedLName")
         userRepository
           .updateUser(updatedUser)
-          .flatMap(
-            _ => userRepository.getUserById(dummyUser.userId).map(dummyUser => dummyUser.get shouldEqual updatedUser)
-          )
+          .flatMap(_ => userRepository.getUserById(dummyUser.userId).map(dummyUser => dummyUser.get shouldEqual updatedUser))
```

Comment:
It's not your flaw, but I think it might be better if you change flatmap style to for comprehensive, they're three cases like this

Reply:
ok i'll try